addons/mail/static/src/messaging/entity/messaging/messaging.js

```diff
@@ -41,6 +42,11 @@ function MessagingFactory({ Entity }) {
             this.update({ isInitialized: true });
         }
 
+        stop() {
+            this.initializer.stop();
+            this.notificationHandler.stop();
+        }
```

Comment:
move `this.env.call('bus_service', 'off', 'window_focus', null, this._handleGlobalWindowFocus);` from delete to here